login.html

```diff
@@ -1,11 +1,61 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>로그인</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    />
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/login_style.css" />
   </head>
   <body>
-    로그인
+    <main>
+      <header>
+        <a href="./index.html">
+          <img src="./img/logo.png" alt="판다마켓 로고" />
+        </a>
+      </header>
+      <form>
+        <label for="email"> 이메일 </label>
```

Comment:
❗️ 수정요청
공백이 들어가면 디자인과 다르게 구현될 가능성이 큽니다~
만약 디자인상 필요한 간격이라면 css로 구현해주시고 불필요한 공백은 제거해주세요!

```suggestion
        <label for="email">이메일</label>
```